Utils/github_workflow_scripts/send_slack_message.py

```diff
@@ -0,0 +1,219 @@
+        {
+            "type": "divider"
+        }
+    ]
+    return pack_details
+
+
+def create_packs_segment(metadata_files: PaginatedList) -> List[dict]:
+    """Aggregate the pack information segments of the message
+
+        Args:
+            metadata_files (PaginatedList): List of File objects representing metadata files
+
+        Returns:
+            (List): List of slack blocks representing all packs information
+    """
+    all_packs = []
+    for file in metadata_files:
+        metadata_obj = get_metadata_file(file)
+        pack_segment = create_individual_pack_segment(metadata_obj)
+        all_packs += pack_segment
+    return all_packs
+
+
+def create_pull_request_segment(pr: PullRequest) -> List[dict]:
+    """Create the pull request information segment of the message
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (List): List containing a slack block-kit section entry which represents the PR info
+    """
+    assignees = ','.join([assignee.login for assignee in pr.assignees])
+    contributor = pr.user.login
+    number_of_changed_changed_files = pr.changed_files
+    labels = ','.join([label.name for label in pr.labels])
+    pr_info_segment = create_slack_fields([
+        ('Assignees', assignees),
+        ('Contributor', contributor),
+        ('Changed Files', number_of_changed_changed_files),
+        ('Labels', labels),
+    ])
+    return [pr_info_segment]
+
+
+def create_pr_title(pr: PullRequest) -> List[dict]:
+    """Create the message title
+
+        Args:
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (List): List containing a dictionary which represents the message title
+    """
+    header = [{
+        "type": "header",
+        "text": {
+            "type": "plain_text",
+            "text": f"{pr.title}",
+            "emoji": True
+        }
+    }]
+    return header
+
+
+def slack_post_message(client: WebClient, message_blocks: List, pr: PullRequest):
+    """Post a message to a slack channel
+
+        Args:
+            client (WebClient): Slack web-client object.
+            message_blocks (List): List of blocks representing the message blocks.
+            pr (PullRequest): object that represents the pull request.
+
+        Returns:
+            (List): List containing a dictionary which represents the message title
+    """
+    client.chat_postMessage(
+        channel=SLACK_CHANNEL_TO_SEND_PR_TO,
+        url="https://google.coms",
+        attachments=[
+            {
+                "color": GREEN_COLOR,
+                "blocks": message_blocks
+            }],
+        text=f"<{pr.html_url}|*New Contribution:* {pr.title}>")
+
+
+def main():
+    t = Terminal()
+    payload_str = get_env_var('EVENT_PAYLOAD')
+    if not payload_str:
+        raise ValueError('EVENT_PAYLOAD env variable not set or empty')
```

Comment:
`get_env_var` raises an `EnvVariableError` if you don't pass a default value as a second argument so there is no need to explicitly raise here.

Reply:
True. Fixed.